Limit TSDiffer comments to one per PR

snowystinger opened the pull request:
Closes <!-- Github issue # here -->

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

<!--- Include instructions to test this pull request -->

## 🧢 Your Project:

<!--- Company/project for pull request -->

LFDanLu reviewed (approved):
logic looks good to me, lets test it. Remind me, what diff does the CI tsdiff produce? Is it a diff of all api changes since last release or between the branch and main?

snowystinger commented:
> logic looks good to me, lets test it. Remind me, what diff does the CI tsdiff produce? Is it a diff of all api changes since last release or between the branch and main?

Diff is currently between branch and main, latest of both. So if you haven't merged main into your branch in a while, this could produce funny diffs for a little bit. But they'd be resolved by merging main in

ktabors reviewed (approved):
Yes please
